File: init/erlang/process.erl
% process.erl
%
-module(process).
-export([hello/0, start/0]).

start() ->
  spawn(?MODULE, hello, []).

hello() ->
  io:format("Hello from ~w~n", [self()]).


% process:hello().
% process:start().
% spawn( fun() -> process:hello() end).

File: init/elixir/process.ex
# process.ex
#
defmodule Process do
  def start do
    spawn(__MODULE__, :hello, [])
  end

  def hello do
    IO.puts("Hello from #{Kernel.inspect(self)}")
  end
end


# % start_link() ->
# %   spawn_link(?MODULE, hello, []).
#
# % process:hello().
# % process:start().
# % spawn( fun() -> process:hello() end).
